README.md

```diff
@@ -1 +1,11 @@
-# posioning_and_floting
\ No newline at end of file
+# Project: Position and floating with CSS3
+
+
+## Description
+This project is about position  different kind of elements  on a web page with the usage of CSS3.
+You can see the live preview on the following url.
+
+## URL
+[Live preview](http://htmlpreview.github.io/?https://github.com/jcromerohdz/posioning_and_floting/blob/master/index.html
```

Comment:
- Your live preview doesn't work.

Reply:
that's because is not int the master at the moment, but it should work ones we pass the challenge

Reply:
Hey thanks for the comment, but unfortunately for us we don't understand what do you mean fixed footer and is cousin frustration for me an my partner, could you be so kind to help us with an example or a resource please.

Reply:
Hi again, I suggest making a fixed footer like the one below (the bottom part where the links with gray color):
![image](https://user-images.githubusercontent.com/26124023/63606737-a55d5880-c5c8-11e9-865f-d689fc7b6723.png)